docs/source/notebooks/regridding.ipynb

```diff
@@ -2744,48 +2724,64 @@
     }
    ],
    "source": [
-    "wfs_url = 'http://boreas.ouranos.ca/geoserver/wfs'\n",
+    "# NBVAL_IGNORE_OUTPUT\n",
+    "\n",
+    "wfs_url = 'http://pavics.ouranos.ca/geoserver/wfs'\n",
```

Comment:
```suggestion
    "wfs_url = 'http://boreas.ouranos.ca/geoserver/wfs'\n",
```

Reply:
Coming out of left field here, but shouldn't we be using `https`?